### PR TITLE
feat(queue): Active executions monitoring

### PR DESCRIPTION
**Pull request by robzienert**
First (well, second) pass at active execution monitoring.

### CODE REVIEW

File: orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+  private fun getActiveExecutions() =
+    pool.resource.use { redis ->
+      redis.hgetAll(REDIS_KEY).map { objectMapper.readValue(it.value, ActiveExecution::class.java) }
+    }
+
+  private fun ActiveExecution.getMetricId() =
+    registry.createId("executions.active")
+      .withTag("type", type.toString())
+      .withTag("application", application)
```
Comment (ajordens) [inline orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt]: I don't think we need application here, having it is going to dramatically up the tag space and may cause issues trying to look historically.

File: orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  }
+
+  private fun getActiveExecutions() =
+    pool.resource.use { redis ->
+      redis.hgetAll(REDIS_KEY).map { objectMapper.readValue(it.value, ActiveExecution::class.java) }
+    }
+
+  private fun ActiveExecution.getMetricId() =
+    registry.createId("executions.active")
+      .withTag("type", type.toString())
```
Comment (ajordens) [inline orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt]: Maybe call this `executionType` to be consistent with other metrics (`task.invocations` at least)

File: orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt (hunk 1 above)
Comment (asher) [reply orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt]: I'd continue to store application in monitor.activeExecutions[execution.id] though, could be useful for /admin functions.

Comment (ajordens) [reply orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt]: Agreed, that's a great call @asher!

**Pull request merged**